crates/viewer/re_view_spatial/tests/tensor_1d.rs

```diff
@@ -0,0 +1,60 @@
+use re_chunk_store::RowId;
+use re_log_types::TimePoint;
+use re_types::{archetypes, datatypes};
+use re_view_spatial::SpatialView2D;
+use re_viewer_context::{ViewClass as _, ViewId, test_context::TestContext};
+use re_viewport::test_context_ext::TestContextExt as _;
+use re_viewport_blueprint::ViewBlueprint;
+
+fn x() -> Vec<f64> {
+    (0..100).map(|i| i as f64 * 100.0 / 99.0).collect()
+}
+
+#[test]
+fn test_tensor() -> anyhow::Result<()> {
```

Comment:
This effectively tests a 1D image that is created froma tensordata struct. Not sure that makes it tensor enough to call it tensor, I think I'd dub this (test & mod) `image_1d` to avoid confusion